test/images/agnhost/Dockerfile

```diff
@@ -55,5 +55,13 @@ ADD agnhost agnhost
 # entrypoint is used by the containers.
 RUN ln -s agnhost agnhost-2
 
+# this user and group is used in a E2E test case of 
+# SupplementalGroups with pre-defined group in the image
+# - image-user(uid=1000)
+# - image-user belongs to group-defined-in-image(gid=50000)
+RUN adduser -u 1000 -D image-user && \
+    addgroup -g 50000 group-defined-in-image && \
+    addgroup image-user group-defined-in-image
```

Comment:
one question: why the different formats for the two names? i would expect both to either  be `image-user & image-group` or `user-defined-in-image & group-defined-in-image`

Reply:
oh, you're right, thanks. I prefer

> `user-defined-in-image & group-defined-in-image`

because this is more specific.

updated in d59af27